scenarios/gaming/README.md

```diff
@@ -0,0 +1,29 @@
+
+A common scenario is to recommend games to a user based on their past interactions with other games. In this scenario, the user is represented by a user profile, and the game is represented by a game profile.  The user profile contains information about the user's past interactions with the game, and the game profile contains information about the game's popularity and other attributes. This scenario is typically shown on the game store homepage (for example, the store of XBox), or it can be send to a user via a personalized newsletter. A large number of algorithms in this repository such as [SAR](../../examples/00_quick_start/sar_movielens.ipynb), [BPR](../../examples/02_model_collaborative_filtering/cornac_bpr_deep_dive.ipynb), and [NCF](../../examples/00_quick_start/ncf_movielens.ipynb) can be used for personalized game recommendation.
+
+### Personalized item recommendation
+
+The addition of micro-transactions inside games have produced a complete shift in the business model of games. Traditionally, most games were based on a one-off purchase, and users got all the content. Nowadays, many games allow players to buy items while they are playing via micro-transactions. Personalized item recommendations can be used to increase the player engagement, tailor the content to the player's interests, and increase the player's retention. In addition, these recommendation provide benefits for studios, which will increase their conversion rate and revenue. These systems also take into account the players past behavior and their interactions with items. The same set of algorithms used in personalized game recommendation can be used for personalized item recommendation, by just adapting the dataset.
+
+### Similar items
+
+A related scenario to personalized item recommendation is similar item recommendations. In this case the recommendations are based on item feature similarity. Some features that can be taken into account are price, item category, item usage, etc. The business usecase for similar items can be increasing the likelihood of buying or avoiding losing a sale by showing an alternative item with a lower price (down-selling). The set of algorithms that can be used in this scenario are those based on content-based filtering, such as [LightGBM](../../examples/00_quick_start/lightgbm_tinycriteo.ipynb), [VW](../../examples/02_model_content_based_filtering/vowpal_wabbit_deep_dive.ipynb) or even [xDeepFM](../../examples/00_quick_start/xdeepfm_criteo.ipynb).
+
+### Next best action prediction
+
+An interesting scenario is next best action prediction. In this scenario, what is recommended is the most beneficial next action for the player. From the technical point of view, this can be implemented using collaborative filtering algorithms, such as [SAR](../../examples/00_quick_start/sar_movielens.ipynb), [BPR](../../examples/02_model_collaborative_filtering/cornac_bpr_deep_dive.ipynb), and [NCF](../../examples/00_quick_start/ncf_movielens.ipynb).

```

Comment:
Isn't this task closer to reinforcement learning rather than CF?

Reply:
Good point, there is a way to do next best action in RL, a lot of people using MCTS. This is done in AlphaGo, AlphaZero, etc. But in that case, the selection of the optimal next action is automatically taken. If we pose the problem as a reco system, where actions are treated like items in the typical CF setup, then the user can choose among a set of actions.

Reply:
I meant something about the state i.e. in CF there is no state. But here is the current state of the game relevant for which next actions to recommend? "Next" implies that you have at least some info about what has happened before. With CF you would always recommend the same action, wouldn't you?

Reply:
The way I see it is like a session based recommender, but with actions instead of items. The info that has happened before are the previous actions, the next recommended action is an option that the user chooses among a list of actions available.